src/MoBi.Presentation/Presenter/EditEventBuilderPresenter.cs

```diff
@@ -112,7 +112,7 @@ public void Edit(EventBuilder eventBuilder, IReadOnlyList<IObjectBase> existingO
 
       public void Edit(EventBuilder eventBuilder)
       {
-         Edit(eventBuilder, eventBuilder.ParentContainer.Children);
+         Edit(eventBuilder, eventBuilder.ParentContainer?.Children);
```

Comment:
We had a couple others like this from another refactoring. I found them while opening editors for each type to check the captions.